draw-sine.lua

```diff
@@ -0,0 +1,51 @@
+-- Aseprite script to draw a sine wave with specified wave frequency and amplitude dampening
+-- Written by Dylmm, 2022
+-- https://github.com/Dylmm/aseprite-scripts
+
+-- Open dialog, ask user for paramters
+function userInput()
+    local dlg = Dialog()
+    -- Create dialog parameters
+    dlg:number{ id="wavefrequency", label="Wave Frequency:", decimals=2, text = "5"}
+    dlg:number{ id="wavedampening", label="Wave Dampening:", decimals=2, text = "1" }
+    dlg:check{ id="fill", label="Fill:", selected=true }
```

Comment:
I'm not sure "Fill" quite makes sense for a sine wave, as it's unclear if that's filling the line or filling under the curve or what. Perhaps change the name of the label.